pkg/util/proto/testdata/swagger.json

```diff
@@ -6,6 +6,621 @@
   },
   "paths": {},
   "definitions": {
+    "io.k8s.apiextensions-apiserver.pkg.apis.apiextensions.v1.CustomResourceColumnDefinition": {
```

Comment:
Was this copied from somewhere or generated from something in that repo?

Reply:
Or did you type all these characters with your keyboard?

Reply:
copied from k/k master swagger.json